allure-generator/src/main/java/io/qameta/allure/allure1/Allure1ResultsReader.java

```diff
@@ -208,6 +208,9 @@ private Attachment convert(ru.yandex.qatools.allure.model.Attachment attachment)
     }
 
     public static Status convert(ru.yandex.qatools.allure.model.Status status) {
+        if (Objects.isNull(status)) {
```

Comment:
this method's body could be simplified:

```java
ofNullable(status).orElse(UNKNOWN);
```

or even easier, if you return `Optional<Status>`